docs/topic_guides/performance/drivers_of_performance.md

```diff
@@ -6,13 +6,30 @@ tags:
 
 This topic guide covers the fundamental drivers of the run time of Splink jobs. 
 
+## :simple-adblock: Blocking
+
 The primary driver of run time is **the number of record pairs that the Splink model has to process**. In Splink, the number of pairs to consider is reduced using **Blocking Rules** which are covered in depth in their own set of [topic guides](../blocking/blocking_rules.md). 
 
-Additional factors which affect performance are:
+## :material-compare: Complexity of comparisons
+
+More complex comparisons reduces performance. Complexity is added to comparisons in a number of ways, including:
+
+- Increasing the number of comparison levels
+- Using more computationally expensive comparison functions
+- Adding Term Frequency Adjustments
+
+!!! note "Performant Term Frequency Adjustments"
+    Model training with Term Frequency adjustments can be made more performant by setting `estimate_without_term_frequencies` parameter to `True` in `estimate_parameters_using_expectation_maximisation`.
+
```

Comment:
It might be obvious, but perhaps it is worth a few words here or elsewhere in this topic guide to highlight the fact that there is usually some kind of competition between performance + accuracy - I feel like it might be easy for users to get caught in the trap of trying to have a very performant model, and then be confused as to why they are not getting good results